packages/manager/modules/enterprise-cloud-database/src/service/details/cluster-size/delete/index.js

```diff
@@ -0,0 +1,38 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+import deleteComponent from './delete.component';
+
+const moduleName = 'ovhManagerEnterpriseCloudDatabaseServiceDetailsClusterSizeDelete';
+
+angular
+  .module(moduleName, [
+    'ui.router',
+  ])
+  .config(/* @ngInject */($stateProvider) => {
+    $stateProvider.state('enterprise-cloud-database.service.details.cluster-size.delete', {
```

Comment:
Should be extracted to a `delete.routing.js` file

Reply:
Done :)